docs: update readme

0hsn opened the pull request.

0hsn committed:
docs: update readme

arifszn commented on README.md:
```diff
@@ -1,17 +1,49 @@
+
+---
+
+### Ask anything
+
+Please use tag `chkware-cli` with the question on [stackoverflow.com](https://stackoverflow.com/questions/tagged/chkware-cli) to ask or discuss anything. Follow the tag if you want to help others with your knowledge.
```
Maybe issue tracker link also?
```markdown
### How do I report feedback and issues?

You can [file an issue](https://github.com/chkware/vscode-ext/issues) on the GitHub repo.
```

arifszn commented on README.md:
```diff
@@ -1,17 +1,49 @@
-- Open command palette by pressing `ctrl` + `shift` + `p`
-- Search for `chkware` commands
-- After code changes, run `Developer: Reload Window` in the new window from command palette
+Get started writing `chkware` configurations with VS Code in three steps:
```
```suggestion
Get started writing `chkware` configurations with VS Code in two steps:
```

arifszn commented:
![Untitled 2](https://user-images.githubusercontent.com/45073703/187076290-ac2d2fad-5335-4cec-90b2-0e5c500d8ebc.gif)

Can this gif be used?

0hsn commented:
> ![Untitled 2](https://user-images.githubusercontent.com/45073703/187076290-ac2d2fad-5335-4cec-90b2-0e5c500d8ebc.gif)
> 
> Can this gif be used?

Yes, good. but, can you check why syntax coloring is not working? It was supposed to set `YAML` syntax, therefore coloring should be there. or i am getting it wrong.

arifszn commented:
> Yes, good. but, can you check why syntax coloring is not working? It was supposed to set `YAML` syntax, therefore coloring should be there. or i am getting it wrong.


As this was already an opened markdown file, the syntax didn't work. But if no opened file were there, it would create a new YAML file. Changing an already opened file's formatting language programmatically isn't possible right now.

![Untitled 5](https://user-images.githubusercontent.com/45073703/187077383-7083de1a-affe-4fed-845a-d71d69d03034.gif)

0hsn commented:
> As this was already an opened markdown file, the syntax didn't work. But if no opened file were there, it would create a new YAML file. Changing an already opened file's formatting language programmatically isn't possible right now.

It's okay if the feature is not available. But, at least the gif looks okay now

0hsn committed:
docs: update readme

- add usage doc
- add report issue link
- update contribution guide and code of conduct link
